System/Clipboard/X11.hs

```diff
@@ -86,14 +93,13 @@ sendSelectionNotify display req sel target prop time = allocaXEvent $ \ev -> do
 stringToChars :: String -> [CUChar]
 stringToChars = map fromIntegral . encode
 
-
-initialSetup :: IO (Display, Window, Atom)
+initialSetup :: IO (Display, Window, [Atom])
 initialSetup = do
     display <- openDisplay ""
     window <- createSimpleWindow display (defaultRootWindow display)
                                  0 0 1 1 0 0 0
-    clipboard <- internAtom display "CLIPBOARD" True
-    return (display, window, clipboard)
+    clipboards <- internAtom display "CLIPBOARD" False
+    return (display, window, [clipboards, pRIMARY])
```

Comment:
Why is this a list instead if it always have 2 elements? Why not extending the tuple?